File: Ej1cond.java
package ej1cond;

import javax.swing.JOptionPane;

class Ej1cond {

	public static void main(String[] args) {
		int a, b, c;
		a = Integer.parseInt(JOptionPane.showInputDialog(null, "ingrese numero"));
		b = Integer.parseInt(JOptionPane.showInputDialog(null, "ingrese numero"));
		c = Integer.parseInt(JOptionPane.showInputDialog(null, "ingrese numero"));
		if (a >= b && a >= c) {
			JOptionPane.showMessageDialog(null, "el mayor es: " + a);
		} else if (b >=a && b >= c){
			JOptionPane.showMessageDialog(null, "el mayor es: " + b);
		} else {
			JOptionPane.showMessageDialog(null, "el mayor es: " + c);
		}
	}

}

File: Ej1cond.js
let a = parseInt(prompt("Ingrese numero"));
let b = parseInt(prompt("Ingrese numero"));
let c = parseInt(prompt("Ingrese numero"));

if (a >= b && a >= c) {
    alert("El mayor es: " + a);
} else if (b >= a && b >= c) {
    alert("El mayor es: " + b);
} else {
    alert("El mayor es: " + c);
}

